pandas/io/common.py

```diff
@@ -153,6 +154,18 @@ def urlopen(*args, **kwargs):
     return urllib.request.urlopen(*args, **kwargs)
 
 
+def is_json(url: FilePathOrBuffer) -> bool:
+    """
+    Returns true if the given string looks like
+    json
+    """
+    json_pattern = re.compile(r"^\s*[\[{]")
+    if json_pattern.match(url):
+        return True
+    else:
+        return False
```

Comment:
can be simplified `return json_pattern.match(url) is not None`

Reply:
thanks, I committed your suggestion